azkaban-exec-server/src/main/java/azkaban/execapp/FlowRunner.java

```diff
@@ -297,6 +306,26 @@ public void run() {
     }
   }
 
+  private void reportFlowFinishedMetrics() {
+    final Status status = this.flow.getStatus();
+    switch (status) {
+      case SUCCEEDED:
+        this.execMetrics.markFlowSuccess();
+        break;
+      case FAILED:
+        this.commonMetrics.markFlowFail();
+        break;
+      case KILLED:
+        this.execMetrics.markFlowKilled();
+        final long now = System.currentTimeMillis();
```

Comment:
tiny nit: I would embed this into the conditional below because (1) we may not even need it, and (2) in case there is a long pause between execution of this and the next line. (Although optimizer may take care of both... I dunno.)

Reply:
Will do.